CDC Service Stream Results

```diff
@@ -0,0 +1,105 @@
+E:\PROJECTS\SPRING\keycloak-cdc\app-service>curl http://127.0.0.1:8082/api/users/stream
+
+
+ADDED NEW USER:
+----------------
+
+id: 76cdec09-c9cf-4372-9350-4b78d122f6b8
+retry: 5000
+data: {"type":"UserAddedToRoleEvent","source":{"user_id":"61d116e4-b25e-4150-a6ca-954841a4dbee","role_id":"1527c402-b55e-4a7d-8f86-fd14f9da7a09"}}
+
+id: 52cbda4a-b885-4feb-817d-9eec4b6a085a
+retry: 5000
+data: {"type":"UserUpdatedEvent","source":{"id":"61d116e4-b25e-4150-a6ca-954841a4dbee","username":"samuel","email":"samjack@gmail.com","enabled":true,"first_name":"Samuel","last_name":"Jackson","created_timestamp":"2020-02-01T06:10:25.760Z","realm_id":"sample"}}
+
+id: 11c50169-555b-40fc-8075-853b53d7fbbf
+retry: 5000
+data: {"type":"UserAddedToRoleEvent","source":{"user_id":"61d116e4-b25e-4150-a6ca-954841a4dbee","role_id":"9845fcba-0918-4460-8dfb-376c6b0684c9"}}
```

Comment:
What's the purpose of this file

Reply:
Oh I was using it as a guide for destructuring the object response I get from kafka in the js.
I wasn't supposed to be in my commit.